src/functions-and-arrays.js

```diff
@@ -1,38 +1,98 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+function maxOfTwoNumbers(number1, number2) {
+  if (number1 < number2) {
+    return number2;
+  } else if (number1 > number2) {
+    return number1;
+  } else {
+    return number1;
+  }
+}
```

Comment:
Esto esta perfect!! pero si te fijas solo tienes un caso para devolver number2 no?
Entonces nos podemos ahorrar una consulta!
```suggestion
function maxOfTwoNumbers(number1, number2) {
  if (number1 < number2) {
    return number2;
  } else {
    return number1;
  }
}
```